tests/helpers/snapshots/test_frame.ambr

```diff
@@ -0,0 +1,120 @@
+# serializer version: 1
+# name: test_report[core default]
+  list([
+  ])
+# ---
+# name: test_report[core integration default]
+  list([
+    "Detected that integration 'test_core_integration' test_report_string at homeassistant/components/test_core_integration/light.py, line 23: self.light.is_on. Please create a bug report at https://github.com/home-assistant/core/issues?q=is%3Aopen+is%3Aissue+label%3A%22integration%3A+test_core_integration%22",
+  ])
+# ---
+# name: test_report[custom integration default]
+  list([
+    "Detected that custom integration 'test_custom_integration' test_report_string at custom_components/test_custom_integration/light.py, line 23: self.light.is_on. Please create a bug report at https://github.com/home-assistant/core/issues?q=is%3Aopen+is%3Aissue+label%3A%22integration%3A+test_custom_integration%22",
```

Comment:
The reason why this (and other tests of custom integrations) ask to create an issue on the core issue tracker is that the `mock_integration_frame` fixture does not patch `sys.modules` which is searched by `frame.get_integration_frame` to get the correct module.

We should improve the `mock_integration_frame` fixture in a follow-up PR.